e2e/cypress/e2e/cclw/journeys/browse.cy.js

```diff
@@ -29,7 +29,7 @@ describe("Browse Flow", () => {
   });
 
   it("should display a list of 20 search results", () => {
-    cy.get(searchResultsSelector).children(searchResultSelector).children({ timeout: 10000 }).should("have.length", 20);
+    cy.get(searchResultsSelector).children(searchResultSelector, { timeout: 20000 }).children().should("have.length", 20);
```

Comment:
Is this to try and fix the flaky test? If so, does it work?

Reply:
Yes, and not sure, but my feeling is it might account for the delay.

I looked at the test itself, you can see in the actions, when it passed here it did so quite quickly, about 1.3 seconds. So I think there is something else affecting it.